t/venom/lib/extract_certificates.yml

```diff
@@ -0,0 +1,52 @@
+executor: extract_certificates
+input:
+  cnt: ""
+  cnCA: ""
+  certPath: ""
```

Comment:
I suggest to keep snake case syntax for variables.

Reply:
Fixed